apps/src/code-studio/components/progress/SelectedStudentInfo.jsx

```diff
@@ -0,0 +1,114 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import i18n from '@cdo/locale';
+import {TeacherPanelProgressBubble} from '@cdo/apps/code-studio/components/progress/TeacherPanelProgressBubble';
+import Button from '@cdo/apps/templates/Button';
+import {LevelStatus} from '@cdo/apps/util/sharedConstants';
+
+const styles = {
+  main: {
+    textAlign: 'center',
+    display: 'flex',
+    justifyContent: 'center',
+    flexDirection: 'column'
+  },
+  bubble: {
+    marginLeft: 81
+  }
+};
+
+export class SelectedStudentInfo extends React.Component {
+  static propTypes = {
+    selectedStudent: PropTypes.object,
+    level: PropTypes.object,
+    inMiniRubricExperiment: PropTypes.bool
+  };
+
+  onUnsubmit = () => {
+    $.ajax({
+      url: `/user_levels/${this.props.level.id}`,
+      method: 'PUT',
+      data: {
+        user_level: {
+          best_result: 1,
```

Comment:
why is best_result being updated to 1 here?

Reply:
This is the way we were implementing these buttons before so I kept it consistent - https://github.com/code-dot-org/code-dot-org/blob/a19e762f8969d905f18e35c7c392e561a0e18ee2/apps/src/code-studio/teacher.js#L45